src/pages/Scholarships.jsx

```diff
@@ -30,44 +30,48 @@ function ScholarshipsPage() {
   const [error, setError] = useState();
   const history = useHistory();
   const location = useLocation();
+  const params = queryString.parse(location.search);
+
+  const sortingOptions = {
+    deadlineSoon: {
+      field: 'deadline',
+      dir: 'asc',
+    },
+    deadlineLatest: {
+      field: 'deadline',
+      dir: 'desc',
+    },
+    amountLow: {
+      field: 'amount.min',
+      dir: 'asc',
+    },
+    amountHigh: {
+      field: 'amount.max',
+      dir: 'desc',
+    },
+  };
 
   const validField = Boolean(
-    queryString.parse(location.search).sortField === 'deadline' ||
-      queryString.parse(location.search).sortField === 'amount.min' ||
-      queryString.parse(location.search).sortField === 'amount.max'
+    params.sortField === 'deadline' ||
+      params.sortField === 'amount.min' ||
+      params.sortField === 'amount.max'
   );
```

Comment:
No need to cast this to a boolean. It's already a boolean.

Reply:
ok changed